### ISSUE TITLE
Minor annoyance in viewSource-content.js

### ISSUE THREAD

**Issue by ThosRTanner** [MoonchildProductions/UXP#1043]
This is in omi.ja!/chrome/toolkit/content/global/viewSource-content.js (which is a one to one copy of  toolkit/components/viewsource/content/viewSource-content.js from the repo).

at line 220 there is this definition

```
/**
   * Called when the parent sends a message to view some source code.
   *
   * @param URL (required)
   *        The URL string of the source to be shown.
   * @param outerWindowID (optional)
   *        The outerWindowID of the content window that has hosted
   *        the document, in case we want to retrieve it from the network
   *        cache.
   * @param lineNumber (optional)
   *        The line number to focus as soon as the source has finished
   *        loading.
   */
  viewSource(URL, outerWindowID, lineNumber) {
```

(3 params)

but it is called at line 130 with

```
    switch (msg.name) {
      case "ViewSource:LoadSource":
        this.viewSource(data.URL, data.outerWindowID, data.lineNumber,
                        data.shouldWrap);
```

(4 params)

So is passing a parameter which is just thrown away. Moreoever, if you have js strict mode warnings enabled, you get complaints every time you click 'view source in debugger' in the debug window that shouldWrap is an undefined property

**Comment by mattatobin** [comment]
You don't know that this is unused elsewhere. Also, please don't waste our time with line references from compiled omnija resources.. Use the cross reference as we preprocess many of those files.

**Comment by wolfbeast** [comment]
To evaluate this, please list the proper source code references.

**Comment by wolfbeast** [comment]
NI @ThosRTanner

**Comment by adeshkp** [comment]
Going through code casually using xref suggests that @ThosRTanner is right and fourth argument is redundant at the call site.

**Comment by ThosRTanner** [comment]
- I thought submitting PRs was effectively asking for a comment on it. If this is not the case, how should I go about this as your contributing.md suggests this is the proper procedure.

- Although there is preprocessing done, those line numbers are a 1-1 match. Clearly I could not do a PR against a file after the preprocessing. However, the *symptoms* are exactly as I have reported them. The web console complains with the line and file number I've quoted.

- The function called does not take a 4th argument, and there is no other javascript function with that name so there is no point in passing it.
